internal/apiserver/clusterservice/clusterimpl.go

```diff
@@ -1356,6 +1356,12 @@ func getClusterParams(request *msgs.CreateClusterRequest, name string, userLabel
 	spec.CustomConfig = request.CustomConfig
 	spec.SyncReplication = request.SyncReplication
 
+	if request.BackrestConfig != "" {
```

Comment:
Thinking we should validate that the ConfigMap provided is valid, like we do for custom PG/Patroni config:

https://github.com/CrunchyData/postgres-operator/blob/6fbd804b57ab68cd4286473ce22b2b1ccf58c25e/internal/apiserver/clusterservice/clusterimpl.go#L682-L683

Reply:
Done!